Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -433,7 +433,7 @@ file that was distributed with this source code.
 
 {% block sonata_type_immutable_array_widget_row %}
     {% spaceless %}
-        <div class="form-group{% if child.vars.errors|length > 0%} error{%endif%}" id="sonata-ba-field-container-{{ id }}-{{ key }}">
+        <div class="form-group{% if child.vars.errors is defined and child.vars.errors|length > 0%} error{%endif%}" id="sonata-ba-field-container-{{ id }}-{{ key }}">
```

Comment:
Might be simplified with `default` ? `if child.vars.errors|default([])|length`. Also, it might be best to ensure errors is always an array on the php side, not sure if this is easy though.